### ISSUE TITLE
Update nginx docs

### ISSUE THREAD

**Issue by johnclary** [codyschank/application#7]
When setting up nginx I thin you need to remove the default config from sites_enabled and sites_available.

**Comment by codyschank** [comment]
Yep. I remember doing that when I followed this tutorial provided by
Insight:
https://docs.google.com/presentation/d/18-PQUJI6BRwi5H8qcgKR_4cBoyj8PTlbKdS4rIREFQs/edit

Starts at slide 40.

**Comment by johnclary** [comment]
Hey, that’s a great slide deck. The sites_available/sights_enabled simlink was already in place, so I guess that happens on install with Ubuntu. In my past experience with nginx (via Docker), this was not the case. I’ll add this stuff to the docs.
